
int8 cam_id
bool streaming
bool plugged
